democracy_club/templates/base.html

```diff
@@ -26,7 +26,7 @@
     <meta property="fb:pages" content="316731041785" />
     <meta property="fb:app_id" content="262795130596272" />
     <!-- Twitter -->
-    <meta name="twitter:title" content="{{ SITE_TITLE }}">
+    <meta name="twitter:title" content="{% block og_title_twitter_content %}{{ SITE_TITLE }}{% endblock og_title_twitter_content %}">
```

Comment:
These are working much better now when testing locally. 

My last comment is that the naming of the blocks is a bit confusing so could be refactored - for example I dont think they should all have the `og_` prefix as they are not related to the `og` meta tags (on lines 17 down).

I'd suggest that each of the blocks more closely mirrors the name of the meta attribute they are related to eg:
```
twitter_title_content
twitter_url
twitter_image_url
twitter_description_content
```